github/github.go

```diff
@@ -880,6 +880,23 @@ func cloneRequest(r *http.Request) *http.Request {
 	return r2
 }
 
+func formatDuration(d time.Duration) string {
```

Comment:
Will take a look into it. Thank for sharing the knowledge about the negative values. I've seen them in the test. But thought it's some kind of obsolete. 
So I'll improve the code to support negative numbers + will write some tests. 

Is it possible to assign the issue to me?